crates/compiler/build/src/program.rs

```diff
@@ -196,7 +196,6 @@ fn gen_from_mono_module_llvm<'a>(
 
     let builder = context.create_builder();
     let (dibuilder, compile_unit) = roc_gen_llvm::llvm::build::Env::new_debug_info(module);
-    let (mpm, _fpm) = roc_gen_llvm::llvm::build::construct_optimization_passes(module, opt_level);
```

Comment:
LLVM optimisation is no longer done at function level, but only at module level.